### ISSUE TITLE
340MB of RAM used is too much

### ISSUE THREAD

**Issue by FezVrasta** [chenasraf/gInbox#22]
Hello

this software should stay open 24h on our Mac, but it uses 340MB of RAM on my MacBook Pro 15" retina mid 2014.

I think is too much, considering that Radiant Player, which uses a similar approach, uses 180MB of ram and has to load Flash and multimedia stuff...

Could be possible improve the RAM impact?

Thanks

**Comment by chenasraf** [comment]
Unfortunately, this is probably Inbox's fault for not clearing up stuff when it should. It also slows down significantly after some time (I often find myself closing & reopening the app just because it's sluggish), Google Music is better at that, I guess. Inbox also draws a lot of bandwidth on the long run, apparently it polls Google's servers every once in a while.

**Comment by FezVrasta** [comment]
I've created a similar app using node-webkit and it uses much less ram, so I don't think is a problem of Inbox.

**Comment by chenasraf** [comment]
I'll do some profiling then.

**Comment by schr3j** [comment]
It also seems to send a lot of data towards google. I just noticed it sent about 9.4MB according to LittleSnitch without sending any emails using it. Is google really recording that much userdata?

**Comment by chenasraf** [comment]
They're polling some scripts very often. Blocking those scripts caused a "Server unavailable" error

**Comment by bilby91** [comment]
~700 mb after a couple of hours with the app open :(

**Comment by fniephaus** [comment]
I think this can be closed, because gInbox won't be able to solve the problem.
I'm hoping the memory consumption will improve over time while Inbox matures...

**Comment by Ke-** [comment]
Agreed, this is a Google issue — memory usage appears to be on par or better than using Inbox within Chrome. Nice work on the wrapper.
